test/mysql_test.cpp

```diff
@@ -100,6 +100,8 @@ TEST_CASE_METHOD(mysql_fixture, "catalog_tables_test", "[mysql][catalog][tables]
     catalog_tables_test();
 }
 
+// TODO: Add catalog_table_privileges_test - SQLTablePrivileges returns empty result set
```

Comment:
I have tried to call `SQLTablePrivileges`  with variety of arguments (e.g. database name, empty database name, `SQL_NTS` or 0 for size, etc.) and this function always returns nothing. I have no idea why.

I'm using MySQL ODBC 5.3 Driver.